src/rtc.rs

```diff
@@ -0,0 +1,480 @@
+//! Interface to the real time clock. See STM32F303 reference manual, section 27.
```

Comment:
Hm... Feels a bit weird to reference the F303 RM here.

Reply:
The original code was written for the F3, I didn't modify those comments.